packages/lief/PKGBUILD

```diff
@@ -1,48 +1,55 @@
 # This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
 # See COPYING for license details.
 
-pkgname=lief
-_pkgname=LIEF
-pkgver=0.13.2
-pkgrel=2
-pkgdesc='Library to Instrument Executable Formats.'
-groups=('blackarch' 'blackarch-disassembler' 'blackarch-reversing')
-arch=('x86_64' 'aarch64')
-url='https://github.com/lief-project/LIEF/'
-license=('Apache')
-depends=('python')
-makedepends=('python-setuptools' 'cmake' 'python-build' 'python-installer'
-             'python-wheel' 'ninja')
-source=("https://github.com/lief-project/LIEF/archive/$pkgver.tar.gz")
-sha512sums=('a8dce9cce9c68d93ec5d78cf9f67eb32dc11d4498c9903f61e34d48683905cdf8040630b390fdaefb66fd18213cdc81682b4acb24ce43fb7fe89202492f71660')
+pkgbase=lief
+pkgname=(lief python-lief)
+pkgver=0.15.1
+pkgrel=1
+pkgdesc='Library to instrument executable formats.'
+arch=('x86_64')
```

Comment:
you removed the blackarch groups here, why? could you also please keep our style? $_pkgname shall be used instead of LIEF

Reply:
Because I used Python lib PKGBUILD template, but you're right. I fixed it.